cmd/opm/alpha/list/cmd.go

```diff
@@ -22,7 +23,17 @@ func NewCmd() *cobra.Command {
 
 ` + humanReadabilityOnlyNote,
 	}
-	list.AddCommand(newPackagesCmd(), newChannelsCmd(), newBundlesCmd())
+	np := newPackagesCmd()
+	np.Flags().Bool("skip-tls-verify", false, "disable TLS verification")
+	np.Flags().Bool("use-http", false, "use plain HTTP")
+	nc := newChannelsCmd()
+	nc.Flags().Bool("skip-tls-verify", false, "disable TLS verification")
+	nc.Flags().Bool("use-http", false, "use plain HTTP")
+	nb := newBundlesCmd()
+	nb.Flags().Bool("skip-tls-verify", false, "disable TLS verification")
+	nb.Flags().Bool("use-http", false, "use plain HTTP")
```

Comment:
It turns out that the `opm index` AddCommand method is adding these global flags (and their validations) to the `opm` parent command.  
I reorganized the code to avoid this indirection, and I deleted these lines as it turns out that we never needed them anyway.  :)

Reply:
Woof.. that means that every single `opm` subcommand will have those flags listed in their help/usage regardless of whether that specific subcommand makes use of them.

That also means that when we remove `opm index` (and we will since it's deprecated), we'll remove the global setup for those flags.

My personal preference would be to explicitly define them on every single subcommand that used them to avoid both of those problems, but that seems outside the scope of this PR, so we could discuss more and follow-up separately.